### PR TITLE
add Clusternet plugin

### PR DESCRIPTION
**Pull request by dixudx**
This PR is a proposal to add the [clusternet](https://github.com/clusternet/clusternet) plugin to krew-index.

[Clusternet](https://github.com/clusternet/clusternet) helps manage multiple Kubernetes clusters and coordinate applications to multiple clusters from a single set of APIs in a hosting cluster.
 
The README in the repos has more information [How It Works](https://github.com/clusternet/kubectl-clusternet#how-it-works).

### CODE REVIEW

**Comment by dixudx** [comment]
@ahmetb PTAL, thx.

File: plugins/clusternet.yaml
Comment (ahmetb) [inline plugins/clusternet.yaml:9]: > A 'kubectl' plugin

this is redundant.

Comment (dixudx) [reply plugins/clusternet.yaml]: addressed

File: plugins/clusternet.yaml
Comment (corneliusweig) [inline plugins/clusternet.yaml:17]: Please use a stable URL here, for example by pointing to HEAD. I understand your motivation to point to the exact docs for this release, however it does not play well with our automated approver bot (which does not like when text fields change).

File: plugins/clusternet.yaml
Comment (corneliusweig) [inline plugins/clusternet.yaml:13]: These usage instructions are shown by default. Please drop.

**Review by corneliusweig — APPROVED** [review]
/lgtm
/approve
/hold

**Comment by dixudx** [comment]
@corneliusweig  Addressed. PTAL. Thx.

**Comment by corneliusweig** [comment]
@dixudx Sorry for the delay. I made a small wording change. Please do `/hold cancel` if you're satisfied.
/lgtm
/approve
/hold

**Pull request merged**
